src/styles/settings/palette.vars.scss

```diff
@@ -0,0 +1,30 @@
+
+[data-palette='sandstone'] {
+  --cdr-color-background-surface-primary: #{$cdr-color-background-secondary};
+  --cdr-color-background-surface-secondary: #dcd6cb;
+  --cdr-color-border-surface-primary: #{$cdr-color-border-secondary};
+  --cdr-color-border-surface-secondary: #726d64;
+}
+
+[data-palette='membership-subtle'] {
+  --cdr-color-background-surface-primary: var(--cdr-membership-subtle-color-background-primary);
```

Comment:
If these get changed to sass variables, solves the double import issue perhaps

Reply:
yes I added sass as a fallback